OloEngine/src/Platform/Linux/LinuxPlatformMemory.cpp

```
@@ -0,0 +1,144 @@
+                    catch (...)
+                    {
+                        return 0;
+                    }
+                    return kb * 1024;
+                }
+            }
+            return 0;
+        }
+    } // namespace
+
+    bool QueryStats(BackendStats& outStats)
```

Cppcheck (2.20.0)

[style] The function 'QueryStats' is never used.

(unusedFunction)